source/index.md

```diff
@@ -26,6 +26,7 @@ includes:
   - methods/collection_requests
   - methods/contacts
   - methods/payments
+  - methods/accounts
```

Comment:
Move this up so that it's at the top? Let's keep the list in Alphabetical order.